verify/.env

```diff
@@ -1,13 +1,4 @@
-# Configure these variables in function configuration:
-# https://www.twilio.com/console/functions/configure
-
-
-# Find these credentials in the Twilio console
-# https://www.twilio.com/console
-ACCOUNT_SID=
-AUTH_TOKEN=
-
-# SID of your Twilio Verify Service
-# https://www.twilio.com/console/verify/services
-# https://www.twilio.com/docs/verify/api/service
+# description: SID of your Twilio Verify Service
+# format: sid
+# link: https://www.twilio.com/console/verify/services
```

Comment:
what about the `required` field?

Reply:
i believe there is a default required value if its not present